nemo_curator/modules/config.py

```diff
@@ -98,3 +98,67 @@ def __post_init__(self):
             raise ValueError("Jaccard Threshold must be between [0,1]")
         if self.buckets_per_shuffle <= 0:
             raise ValueError("Buckets per shuffle must be greater than 0")
+
+
+@dataclass
+class SemDedupConfig(BaseConfig):
+    """
+    Configuration for Semantic Deduplication.
+
+    Attributes:
+        cache_dir (str): Directory to store cache.
+        num_files (int): Number of files. Default is -1, meaning all files.
+        id_col_name (str): Column name for ID.
+        id_col_type (str): Column type for ID.
+        input_column (str): Input column for embeddings.
+        input_file_type (str): File type for input embeddings.
+        embeddings_save_loc (str): Location to save embeddings.
+        model_name_or_path (str): Model name or path for embeddings.
+        batch_size (int): Inital Batch size for processing embeddings.
```

Comment:
batch_size should be embeddings_batch_size

Reply:
Resolved by https://github.com/NVIDIA/NeMo-Curator/pull/130/commits/52480aa4f3dbeb5761a18e5c159591f881dc4a79